script.js

```diff
@@ -12,6 +12,116 @@ const headerListElement = document.getElementById('header_list');
 const buttonElement = document.getElementById('button_back');
 let menuIsActive = false;
 let submenuIsActive = false;
+const buttonRight = document.getElementById('button_right');
+const buttonLeft = document.getElementById('button_left');
+const carouselText = document.getElementById('carousel_text');
+const carouselBox = document.getElementById('carousel_box');
+const carouselElement = document.getElementById('carousel');
+
+let carouselTextObject = document.createDocumentFragment();
+carouselTextObject = carouselText;
+
+let images = Array.from(carouselBox.children);
+let slider = [];
+images.forEach(function(item, i, images) {
```

Comment:
What is i? And why are you passing images to this function?

Reply:
Deleted i .
I want to collect the paths (src) to all images (from array images);